src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-pypemicro.py

```diff
@@ -0,0 +1,35 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2022 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+
+# Hook for the pypemicro module: https://github.com/nxpmicro/pypemicro
+
+import glob
+import os
+from PyInstaller import isolated
+from PyInstaller.utils.hooks import remove_prefix, get_package_paths
+
+
+@isolated.decorate
+def get_safe_libs():
+    from pypemicro import PyPemicro
+    libs = PyPemicro.get_pemicro_lib_list()
+    return libs
+
+
+pkg_base, pkg_dir = get_package_paths("pypemicro")
+
+binaries = list()
+for lib in get_safe_libs():
+    source = lib['path']
+    for file in glob.glob(os.path.join(source, '*')):
+        dest = remove_prefix(source, os.path.dirname(pkg_base) + os.sep)
```

Comment:
Use `os.path.relpath` instead of `remove_prefix`. That way you don't need to worry about trailing separators.

Reply:
done